codegen/src/utils/fetch_metadata.rs

```diff
@@ -62,7 +62,7 @@ pub fn fetch_metadata_hex_blocking(
 
 // Block on some tokio runtime for sync contexts
 fn tokio_block_on<T, Fut: std::future::Future<Output = T>>(fut: Fut) -> T {
-    tokio::runtime::Builder::new_multi_thread()
+    tokio::runtime::Builder::new_current_thread()
```

Comment:
can you explain this change? :)

Reply:
Oh yea, that shouldn't change! Initially, tokio was included with features in the workspace that did not play nice with wasm environment. Since then I've changed the tokio dependency to not include features and forgot about this! Thanks for spotting it!